ui/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+  plugins: [react()],
+  envPrefix: "CTRON_",
```

Comment:
Ahh I see, I was going to ask "dont we have to prefix with `VITE_`". Do you mind if we keep it with the `VITE_` prefix? Even if longer here and there, I think its good the external consistency with most other vite apps by default.

Reply:
feel free to change it, but I think using something like `CTRON_` for everything will help us consolidate env vars in the future. If you want to go back to `VITE_`, just remove L6.